features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+
+    Scenario: Core Category Section with icon only
+        Given the side navbar is in the starting state
+        Then SaaSquatch Product News are displayed as the last menu item without an arrow
+        When the user clicks on this item
+        Then a home page for Product News opens up
+        And there are no sub-categories for this item type
+
+    Scenario: The navigation item with an arrow down opens a dropdown menu
+        Given the user selected a core category
+        When the user clicks on a "navigation item with an arrow down"
+        Then the list of articles are shown under the item
+        And there could be further dropdown items between the articles
+        But the page is not changing
+
+    Scenario: The navigation item with an arrow up closes a dropdown menu
+        Given the user selected a "navigation item with an arrow down"
+        When the user clicks on a "navigation item with an arrow up"
+        Then they see the list of articles are closed
+        And the arrow changes back to down
+
+    Scenario: The article item in the menu opens up an article
+        Given the user opened up a "navigation item with an arrow down"
+        When the user clicks on the "article" item
+        Then the article opens up in the central section of the page
+
+    Scenario: The core category item at the top of the navbar with an icon stays there until you are at that category and used to navigate back to the home page of this category
+        Given the user navigates within a core category
+        Then they see that the "Core category item at the top of the navbar with an icon" item is visible all the time
+        When they click on this item from anywhere withing the category
+        Then the home page opens up for this core category
+        And it changes to a selected state until the user is on the home page
+
+    Scenario: The navbar items within a dropdown could be separated into further categories by a line and title
+        Given the user opens up a "navigation item with an arrow down"
+        Then they see articles separated by titles
+        And a grey line ("example Developer Resources/API")
+        And this separator item is not clickable
+
+    Scenario: The breadcrumb at the top of the side navbar can be used to navigate back to the starting view of the side navbar and shows 3 levels of navigation
+        Given the user is on the docs page
+        Then the breadcrumb is at the top of the navbar under the search bar
+        And shows the current location of the user in the navbar 3 levels deep
+        And the order of items are "Menu/Core Category/Sub-category"
+
+    Scenario: Special article menu items for API methods
+        Given the user looks at the "Developer REsources/API" section
+        When they open up a methods
+        Then they see button like tags used to highlight method characteristics (example:"https://docs.saasquatch.com/api/methods#get_account")
+
+
```

Comment:
Missing spec: Most of the specs are about users navigating and working with the sidebar. What happens to the sidebar when they drop into an article by clicking a link, search result, home page card etc - does it react and update?